GameData/ROCapsules/PartConfigs/MercuryBDB/MercuryCM.cfg

```diff
@@ -0,0 +1,528 @@
+		affectFARVoxels = True
+
+		SUBTYPE
+		{
+			name = Window
+			transform = Window
+		}
+
+		SUBTYPE
+		{
+			name = Portholes
+			transform = Portholes
+		}
+
+	}
+
+
+	MODULE
+	{
+		name = ModuleB9PartSwitch
+		moduleID = meshSwitchPaint
+		switcherDescription = Paint
+		switcherDescriptionPlural = Paints
+		affectDragCubes = True
+		affectFARVoxels = True
+		SUBTYPE
+		{
+			name = Black
+			transform = Black
+		}
+		SUBTYPE
+		{
+			name = White
+			transform = Black
+			TEXTURE
+			{
+				texture = ROCapsules/Assets/BDB/Mercury/bluedog_Mercury_AlternateTextures
+				baseTransform = Black
+				transform = Skin
+			}
+		}
+		SUBTYPE
+		{
+			name = Mixed
+			transform = Mixed
+		}
+		SUBTYPE
+		{
+			name = Boilerplate
+			transform = Orange
+			TEXTURE
+			{
+				texture = ROCapsules/Assets/BDB/Mercury/bluedog_Mercury_AlternateTextures
+				transform = Skin
+			}
+		}
+	}
+	
+	MODULE
+	{
+		name = ModuleB9AssignUiGroups
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleRCSFX
+				thrusterTransformName = rcsTransform
+			}
+			uiGroupName = rcsRoll
+			uiGroupDisplayName = RCS Roll
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleCommand
+			}
+			uiGroupName = command
+			uiGroupDisplayName = Command
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleReactionWheel
+			}
+			uiGroupName = internal
+			uiGroupDisplayName = Internal
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleDataTransmitter
+				antennaType = INTERNAL
+			}
+			uiGroupName = internal
+			uiGroupDisplayName = Internal
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleScienceExperiment
+				experimentID = crewReport
+			}
+			uiGroupName = command
+			uiGroupDisplayName = Command
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleScienceContainer
+			}
+			uiGroupName = command
+			uiGroupDisplayName = Command
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleAnimateGeneric
+				animationName = deploy_periscope
+			}
+			uiGroupName = command
+			uiGroupDisplayName = Command
+		}
+		MODULE
+		{
+			IDENTIFIER
+			{
+				name = ModuleColorChanger
+			}
+			uiGroupName = command
+			uiGroupDisplayName = Command
+		}
+	}
+}
+@PART[ROC-MercuryCM]:FOR[ROCapsules]:NEEDS[TacLifeSupport]
+{
+	@description ^= :$: Supports a crew of one for up to maximum 35 hours of active operations but it will require you to change the resources.
+
+	@MODULE[ModuleFuelTanks]
+	{
+		@volume += 61
+
+		TANK
+		{
+			name = Food
+			amount = 0
+			maxAmount = 8.5302
+		}
+
+		TANK
+		{
+			name = Water
+			amount = 0
+			maxAmount = 5.6448
+		}
+
+		TANK
+		{
+			name = Oxygen
+			amount = 0
+			maxAmount = 863.1
+		}
+
+		TANK
+		{
+			name = LithiumHydroxide
+			amount = 1.05
+			maxAmount = 1.05
+		}
+
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 7.1820
+		}
+
+		TANK
+		{
+			name = Waste
+			amount = 0
+			maxAmount = 5.7305
+		}
+
+		TANK
+		{
+			name = CarbonDioxide
+			amount = 0
+			maxAmount = 380
+		}
+	}
```

Comment:
We *really* need a generic LifeSupport pass, that can work like the heatshield pass:  Set a flag, request a duration, request number of personnel, let the tool add these to the tank and create the converter.